### ISSUE TITLE
Cannot modify resources such as CPU, RAM on Docker Desktop Edge

### ISSUE THREAD

**Issue by jgan129** [docker/for-win#5684]
- [x] I have tried with the latest version of my channel ( Edge 2.1.7.0)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: E1EF7C28-29A1-4C2F-A7F7-8635015B2FFE/20200204075908

### Expected behavior
Docker restart should take place and Docker VM should have the new settings regarding CPU/RAM

### Actual behavior
New CPU/RAM settings are not applied and restored to the original values.
### Information

  - Windows Version: Windows 10
  - Docker Desktop Version: (Edge 2.1.7.0 - 41536)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No, I am running on a physical machine.

### Steps to reproduce the behavior

Problem is reproducible after updating from latest stable to latest Edge version with the following steps:
1. Open settings of Docker Desktop Edge (e.g. system tray)
2. Navigate to Resources -> Advanced
3. Increase either CPU or Memory
4. Press "Apply & Restart"
5. Close Settings window
6. Open and navigate back to settings -> advanced -> resources. Everything has reset back to the original values

[log.txt](https://github.com/docker/for-win/files/4152140/log.txt)
This log.txt file contains gives a snapshot at the time CPU/RAM increased the "Apply & Restart" Button pressed

**Comment by mat007** [comment]
```
[10:00:58.219][APIRequestLogger  ][Info   ] [398c6ce4] <Server end> POST http://null/settings -> 500 InternalServerError (took 35ms)
```
As you can see there isn’t much information when it fails, since `2.1.7.0` we’ve added a log here.
You can wait for the next release (Stable or Edge) or try this internal private build of Edge https://download-stage.docker.com/win/edge/42598/Docker%20Desktop%20Installer.exe to trigger the error being logged.

**Comment by jgan129** [comment]
Thanks @mat007 , this private build works perfectly now!

**Comment by stephen-turner** [comment]
From the last comment, it sounds like this is fixed in the latest release (2.2.0.3), so closing the ticket.
